Sources/RxAlamofire.swift

```diff
@@ -784,6 +790,25 @@ extension Reactive where Base: DataRequest {
         }
     }
 
+    public func responseJSON() -> Observable<DataResponse<Any>> {
+        return Observable.create { observer in
+            let request = self.base
+
+            request.responseJSON { response in
+                if let error = response.result.error {
```

Comment:
I also contemplated with that. But if we skip the error part and let the request push the error, you would end up with both an error and a next event (next sent first along with an error). I had it set up this way but wasn't sure if it makes sense, I'm down changing back if it makes more sense but I'm worried on the underlying possible confusion.